### ISSUE TITLE
When using native-shim.js element instanceof HTMLElement returns false for native html elements

### ISSUE THREAD

**Issue by jaroslawsawicki** [webcomponents/custom-elements#32]
Steps to reproduce:
1. Load `native-shim.js` 
2. Observe output
```javascript
const element = document.createElement('div');
console.log(element instanceof HTMLElement); //output false
```

**Comment by valdrinkoshi** [comment]
Duplicate of https://github.com/webcomponents/webcomponentsjs/issues/623

**Comment by bicknellr** [comment]
Fixed as of v1.0.0-alpha.4: http://jsbin.com/revequq/edit?html,console,output

**Comment by bicknellr** [comment]
The way the new polyfill handles this is by setting the prototype of the replacement HTMLElement constructor to the old one's prototype. `x instanceof T` is effectively a recursive walk of `x`'s prototype chain to find `T.prototype`, so this ends up working for both custom and built-in elements.
